<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="T4ublrlpusr093" name="scripts.CartOperTest" tests="196" failures="8" timestamp="08 Jan 2019 12:31:50 GMT" time="1041.587" errors="3">
  <testcase name="navigatetoregister" time="2.477" classname="scripts.CartOperTest"/>
  <testcase name="validateNameofBuyer" time="5.490" classname="scripts.CartOperTest"/>
  <testcase name="validateAddress" time="8.990" classname="scripts.CartOperTest"/>
  <testcase name="validateMobile" time="9.084" classname="scripts.CartOperTest"/>
  <testcase name="validateInvalidMobile" time="7.024" classname="scripts.CartOperTest"/>
  <testcase name="validatewithoutOTP" time="6.974" classname="scripts.CartOperTest"/>
  <testcase name="validatewithInvalidOTP" time="6.977" classname="scripts.CartOperTest"/>
  <testcase name="validatevillage" time="6.994" classname="scripts.CartOperTest"/>
  <testcase name="validateWithOutPinCode" time="6.931" classname="scripts.CartOperTest"/>
  <testcase name="validateWithInvalidPinCode" time="6.925" classname="scripts.CartOperTest"/>
  <testcase name="validateWithoutUserName" time="6.891" classname="scripts.CartOperTest"/>
  <testcase name="validateWithoutpwd" time="6.954" classname="scripts.CartOperTest"/>
  <testcase name="validateWithinvalidpwd" time="6.913" classname="scripts.CartOperTest"/>
  <testcase name="validateWithoutcnfmpwd" time="6.948" classname="scripts.CartOperTest"/>
  <testcase name="validateWithMismatchcnfmpwd" time="8.594" classname="scripts.CartOperTest"/>
  <testcase name="validateOtpMismatch" time="10.299" classname="scripts.CartOperTest"/>
  <testcase name="registerWithValidUser" time="9.050" classname="scripts.CartOperTest"/>
  <testcase name="zfillValidData" time="15.210" classname="scripts.CartOperTest"/>
  <testcase name="verifyUser" time="0.002" classname="scripts.CartOperTest"/>
  <testcase name="verifypincode" time="0.002" classname="scripts.CartOperTest"/>
  <testcase name="yavalidateUsernameAlready" time="28.368" classname="scripts.CartOperTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at generic.MainActivity.asserAlert(MainActivity.java:94)
at scripts.RegisterAsIndividual.yavalidateUsernameAlready(RegisterAsIndividual.java:431)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- yavalidateUsernameAlready -->
  <testcase name="verifyMobile" time="0.000" classname="scripts.CartOperTest"/>
  <testcase name="xvalidateMobileAlready" time="7.009" classname="scripts.CartOperTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at generic.MainActivity.asserAlert(MainActivity.java:94)
at scripts.RegisterAsIndividual.xvalidateMobileAlready(RegisterAsIndividual.java:399)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- xvalidateMobileAlready -->
  <testcase name="verifyNameWithDB" time="0.004" classname="scripts.CartOperTest"/>
  <testcase name="wNavigateToRegister" time="2.758" classname="scripts.CartOperTest"/>
  <testcase name="verifyvillage" time="0.001" classname="scripts.CartOperTest"/>
  <testcase name="zNavigateToLogin" time="2.331" classname="scripts.CartOperTest"/>
  <testcase name="verifygst" time="0.002" classname="scripts.CartOperTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at pages.RegisterPage.getGSTNO(RegisterPage.java:115)
at scripts.RegisterAsIndividual.verifygst(RegisterAsIndividual.java:367)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- verifygst -->
  <testcase name="aDBSet" time="6.668" classname="scripts.CartOperTest"/>
  <testcase name="verifyAddress" time="0.006" classname="scripts.CartOperTest"/>
  <testcase name="verifyemail" time="0.000" classname="scripts.CartOperTest"/>
  <testcase name="verifypan" time="0.003" classname="scripts.CartOperTest"/>
  <testcase name="navigateToForgotPwd" time="5.261" classname="scripts.CartOperTest"/>
  <testcase name="forgotPasswordwithoutUserName" time="5.574" classname="scripts.CartOperTest"/>
  <testcase name="forgotPasswordwithoutNum" time="6.924" classname="scripts.CartOperTest"/>
  <testcase name="forgotPasswordwithInvalidMobile" time="8.369" classname="scripts.CartOperTest"/>
  <testcase name="forgotPasswordwithInvaliddata" time="11.197" classname="scripts.CartOperTest">
    <error type="org.openqa.selenium.NoSuchElementException" message="no such element: Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;//*[@class=&#039;swal-text&#039;]&quot;}
  (Session info: chrome=71.0.3578.98)
  (Driver info: chromedriver=2.42.591088 (7b2b2dca23cca0862f674758c9a3933e685c27d5),platform=Windows NT 10.0.16299 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;3.14.0&#039;, revision: &#039;aacccce0&#039;, time: &#039;2018-08-02T20:13:22.693Z&#039;
System info: host: &#039;T4UBLRLPUSR093&#039;, ip: &#039;172.16.2.150&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_181&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.42.591088 (7b2b2dca23cca0..., userDataDir: C:\Users\CGN04\AppData\Loca...}, cssSelectorsEnabled: true, databaseEnabled: false, goog:chromeOptions: {debuggerAddress: localhost:50526}, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 71.0.3578.98, webStorageEnabled: true}
Session ID: c3d7329af8c736bcf6e058926038efc9
*** Element info: {Using=xpath, value=//*[@class=&#039;swal-text&#039;]}">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@class='swal-text']"}
  (Session info: chrome=71.0.3578.98)
  (Driver info: chromedriver=2.42.591088 (7b2b2dca23cca0862f674758c9a3933e685c27d5),platform=Windows NT 10.0.16299 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:13:22.693Z'
System info: host: 'T4UBLRLPUSR093', ip: '172.16.2.150', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_181'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.42.591088 (7b2b2dca23cca0..., userDataDir: C:\Users\CGN04\AppData\Loca...}, cssSelectorsEnabled: true, databaseEnabled: false, goog:chromeOptions: {debuggerAddress: localhost:50526}, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 71.0.3578.98, webStorageEnabled: true}
Session ID: c3d7329af8c736bcf6e058926038efc9
*** Element info: {Using=xpath, value=//*[@class='swal-text']}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)
at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)
at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:322)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:424)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:314)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy13.getText(Unknown Source)
at generic.MainActivity.getAlertText(MainActivity.java:261)
at generic.MainActivity.asserAlert(MainActivity.java:90)
at scripts.ForgotPassword.forgotPasswordwithInvaliddata(ForgotPassword.java:73)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- forgotPasswordwithInvaliddata -->
  <testcase name="forgotPasswordwithValidData" time="4.993" classname="scripts.CartOperTest"/>
  <testcase name="forgotPasswordwithoutOTP" time="8.575" classname="scripts.CartOperTest"/>
  <testcase name="forgotPasswordwithInvalidOTP" time="6.913" classname="scripts.CartOperTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at generic.MainActivity.asserAlert(MainActivity.java:94)
at scripts.ForgotPassword.forgotPasswordwithInvalidOTP(ForgotPassword.java:99)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- forgotPasswordwithInvalidOTP -->
  <testcase name="forgotPasswordwithoutPWD" time="7.214" classname="scripts.CartOperTest"/>
  <testcase name="forgotPasswordwithinvalidPWD" time="8.178" classname="scripts.CartOperTest"/>
  <testcase name="forgotPasswordwithoutCnfmPWD" time="8.283" classname="scripts.CartOperTest"/>
  <testcase name="forgotPasswordwithMissmatchPWD" time="9.662" classname="scripts.CartOperTest"/>
  <testcase name="forgotPasswordwithWrongOTP" time="12.484" classname="scripts.CartOperTest">
    <error type="org.openqa.selenium.NoSuchElementException" message="no such element: Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;//*[@class=&#039;swal-text&#039;]&quot;}
  (Session info: chrome=71.0.3578.98)
  (Driver info: chromedriver=2.42.591088 (7b2b2dca23cca0862f674758c9a3933e685c27d5),platform=Windows NT 10.0.16299 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;3.14.0&#039;, revision: &#039;aacccce0&#039;, time: &#039;2018-08-02T20:13:22.693Z&#039;
System info: host: &#039;T4UBLRLPUSR093&#039;, ip: &#039;172.16.2.150&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_181&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.42.591088 (7b2b2dca23cca0..., userDataDir: C:\Users\CGN04\AppData\Loca...}, cssSelectorsEnabled: true, databaseEnabled: false, goog:chromeOptions: {debuggerAddress: localhost:50526}, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 71.0.3578.98, webStorageEnabled: true}
Session ID: c3d7329af8c736bcf6e058926038efc9
*** Element info: {Using=xpath, value=//*[@class=&#039;swal-text&#039;]}">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@class='swal-text']"}
  (Session info: chrome=71.0.3578.98)
  (Driver info: chromedriver=2.42.591088 (7b2b2dca23cca0862f674758c9a3933e685c27d5),platform=Windows NT 10.0.16299 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:13:22.693Z'
System info: host: 'T4UBLRLPUSR093', ip: '172.16.2.150', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_181'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, acceptSslCerts: false, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.42.591088 (7b2b2dca23cca0..., userDataDir: C:\Users\CGN04\AppData\Loca...}, cssSelectorsEnabled: true, databaseEnabled: false, goog:chromeOptions: {debuggerAddress: localhost:50526}, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 71.0.3578.98, webStorageEnabled: true}
Session ID: c3d7329af8c736bcf6e058926038efc9
*** Element info: {Using=xpath, value=//*[@class='swal-text']}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)
at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)
at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:322)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:424)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:314)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy13.getText(Unknown Source)
at generic.MainActivity.getAlertText(MainActivity.java:261)
at generic.MainActivity.asserAlert(MainActivity.java:90)
at scripts.ForgotPassword.forgotPasswordwithWrongOTP(ForgotPassword.java:162)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- forgotPasswordwithWrongOTP -->
  <testcase name="forgotPasswordwithAllValidDATA" time="14.682" classname="scripts.CartOperTest"/>
  <testcase name="loginWithoutInput" time="10.569" classname="scripts.CartOperTest"/>
  <testcase name="loginWithUsername" time="6.392" classname="scripts.CartOperTest"/>
  <testcase name="loginWithPwd" time="6.544" classname="scripts.CartOperTest"/>
  <testcase name="validCredential" time="5.058" classname="scripts.CartOperTest"/>
  <testcase name="validateLocationTypoHead" time="2.791" classname="scripts.CartOperTest"/>
  <testcase name="validateWithoutDistrict" time="5.487" classname="scripts.CartOperTest"/>
  <testcase name="validateWithoutStoskYard" time="7.562" classname="scripts.CartOperTest"/>
  <testcase name="validateWithoutQTY" time="15.502" classname="scripts.CartOperTest"/>
  <testcase name="validateDistrict" time="0.800" classname="scripts.CartOperTest"/>
  <testcase name="validateStockyard" time="20.169" classname="scripts.CartOperTest"/>
  <testcase name="validateAvailableStock_and_uniPrice" time="13.245" classname="scripts.CartOperTest"/>
  <testcase name="valdtBookingForm" time="1.638" classname="scripts.CartOperTest"/>
  <testcase name="valdtBookingForm" time="1.806" classname="scripts.CartOperTest"/>
  <testcase name="valdtBookingForm" time="1.786" classname="scripts.CartOperTest"/>
  <testcase name="valdtBookingForm" time="1.787" classname="scripts.CartOperTest"/>
  <testcase name="valdtBookingForm" time="1.800" classname="scripts.CartOperTest"/>
  <testcase name="valdtBookingForm" time="1.753" classname="scripts.CartOperTest"/>
  <testcase name="valdtBookingForm" time="1.813" classname="scripts.CartOperTest"/>
  <testcase name="valdtBookingForm" time="1.755" classname="scripts.CartOperTest"/>
  <testcase name="valdtBookingForm" time="1.829" classname="scripts.CartOperTest"/>
  <testcase name="exeedUserlimit" time="1.233" classname="scripts.CartOperTest"/>
  <testcase name="checktotalQty" time="0.090" classname="scripts.CartOperTest"/>
  <testcase name="zReset" time="0.001" classname="scripts.CartOperTest"/>
  <testcase name="dValidatePrice" time="0.485" classname="scripts.CartOperTest"/>
  <testcase name="vlidtBookFrm" time="4.103" classname="scripts.CartOperTest"/>
  <testcase name="vlidtBookFrm" time="4.115" classname="scripts.CartOperTest"/>
  <testcase name="vlidtBookFrm" time="4.135" classname="scripts.CartOperTest"/>
  <testcase name="vlidtBookFrm" time="4.122" classname="scripts.CartOperTest"/>
  <testcase name="vlidtBookFrm" time="4.068" classname="scripts.CartOperTest"/>
  <testcase name="vlidtBookFrm" time="4.129" classname="scripts.CartOperTest"/>
  <testcase name="vlidtBookFrm" time="4.126" classname="scripts.CartOperTest"/>
  <testcase name="vlidtBookFrm" time="4.121" classname="scripts.CartOperTest"/>
  <testcase name="vlidtBookFrm" time="1.726" classname="scripts.CartOperTest"/>
  <testcase name="aCheckPrices" time="2.146" classname="scripts.CartOperTest"/>
  <testcase name="navigateToReviewBooking" time="3.427" classname="scripts.CartOperTest"/>
  <testcase name="aCheckQty" time="2.077" classname="scripts.CartOperTest"/>
  <testcase name="validateRBWithoutLocation" time="5.589" classname="scripts.CartOperTest"/>
  <testcase name="validateRBWithoutAltnatMobile" time="8.206" classname="scripts.CartOperTest"/>
  <testcase name="validateRBInvalidAltnatMobile" time="9.770" classname="scripts.CartOperTest"/>
  <testcase name="validateRBWithValidData" time="7.818" classname="scripts.CartOperTest"/>
  <testcase name="validateStockYardinCP" time="0.046" classname="scripts.CartOperTest"/>
  <testcase name="validateUnitPriceinCP" time="0.048" classname="scripts.CartOperTest"/>
  <testcase name="validateTotalQtyinCP" time="0.072" classname="scripts.CartOperTest"/>
  <testcase name="validateTotalPriceinCP" time="0.046" classname="scripts.CartOperTest"/>
  <testcase name="validateGstinCP" time="0.050" classname="scripts.CartOperTest"/>
  <testcase name="validateTotalPayableinCP" time="0.085" classname="scripts.CartOperTest"/>
  <testcase name="checkReviewBookingBackward" time="0.247" classname="scripts.CartOperTest">
    <failure type="java.lang.AssertionError" message="expected [1, Mahatma Gandhi Main Road, Upper Bazar, Ranchi, Jharkhand] but found [2345678909]">
      <![CDATA[java.lang.AssertionError: expected [1, Mahatma Gandhi Main Road, Upper Bazar, Ranchi, Jharkhand] but found [2345678909]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at scripts.ConsumerModel.checkReviewBookingBackward(ConsumerModel.java:435)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkReviewBookingBackward -->
  <testcase name="checkalternateMobileBackward" time="0.595" classname="scripts.CartOperTest">
    <failure type="java.lang.AssertionError" message="expected [2345678909] but found []">
      <![CDATA[java.lang.AssertionError: expected [2345678909] but found []
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at scripts.ConsumerModel.checkalternateMobileBackward(ConsumerModel.java:445)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkalternateMobileBackward -->
  <testcase name="checkBookFormBackward" time="2.132" classname="scripts.CartOperTest"/>
  <testcase name="checkBookFormBackward" time="2.127" classname="scripts.CartOperTest"/>
  <testcase name="checkBookFormBackward" time="2.097" classname="scripts.CartOperTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at scripts.ConsumerModel.asserCft(ConsumerModel.java:636)
at scripts.ConsumerModel.checkTotlcft(ConsumerModel.java:576)
at scripts.ConsumerModel.checkBookFormBackward(ConsumerModel.java:452)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkBookFormBackward -->
  <testcase name="checkBookFormBackward" time="2.127" classname="scripts.CartOperTest"/>
  <testcase name="checkBookFormBackward" time="2.113" classname="scripts.CartOperTest"/>
  <testcase name="checkBookFormBackward" time="2.135" classname="scripts.CartOperTest"/>
  <testcase name="checkBookFormBackward" time="2.113" classname="scripts.CartOperTest"/>
  <testcase name="checkBookFormBackward" time="2.140" classname="scripts.CartOperTest"/>
  <testcase name="checkBookFormBackward" time="2.111" classname="scripts.CartOperTest"/>
  <testcase name="verifyTotalQTY" time="0.036" classname="scripts.CartOperTest"/>
  <testcase name="verifyPrice" time="0.149" classname="scripts.CartOperTest">
    <failure type="java.lang.AssertionError" message="expected [1.0] but found [0.0]">
      <![CDATA[java.lang.AssertionError: expected [1.0] but found [0.0]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:442)
at scripts.ConsumerModel.verifyPrice(ConsumerModel.java:481)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyPrice -->
  <testcase name="forwardtoConfirmPage" time="2.348" classname="scripts.CartOperTest"/>
  <testcase name="navigateToPayment" time="11.260" classname="scripts.CartOperTest"/>
  <testcase name="validatePaymentPrice" time="0.058" classname="scripts.CartOperTest"/>
  <testcase name="cancelPayment" time="19.960" classname="scripts.CartOperTest"/>
  <testcase name="paymentFail" time="0.775" classname="scripts.CartOperTest"/>
  <testcase name="checkBookingid" time="0.056" classname="scripts.CartOperTest"/>
  <testcase name="checkPaymentStatus" time="0.054" classname="scripts.CartOperTest"/>
  <testcase name="checkAddress" time="0.054" classname="scripts.CartOperTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at scripts.Payment.addressCheck(Payment.java:119)
at scripts.Payment.checkAddress(Payment.java:110)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkAddress -->
  <testcase name="zzRes" time="1.231" classname="scripts.CartOperTest"/>
  <testcase name="zBookingForm" time="20.903" classname="scripts.CartOperTest"/>
  <testcase name="navigateToHome" time="6.352" classname="scripts.CartOperTest"/>
  <testcase name="znavigateToPayPage" time="20.317" classname="scripts.CartOperTest"/>
  <testcase name="navigateToPaymentPage" time="11.824" classname="scripts.CartOperTest"/>
  <testcase name="navigateToPaymentPage" time="11.768" classname="scripts.CartOperTest"/>
  <testcase name="navigateToPaymentPage" time="11.814" classname="scripts.CartOperTest"/>
  <testcase name="navigateToPaymentPage" time="11.824" classname="scripts.CartOperTest"/>
  <testcase name="navigateToPaymentPage" time="11.797" classname="scripts.CartOperTest"/>
  <testcase name="navigateToPaymentPage" time="11.821" classname="scripts.CartOperTest"/>
  <testcase name="navigateToPaymentPage" time="11.828" classname="scripts.CartOperTest"/>
  <testcase name="navigateToPaymentPage" time="11.761" classname="scripts.CartOperTest"/>
  <testcase name="navigateToPaymentPage" time="12.162" classname="scripts.CartOperTest"/>
  <testcase name="aCreditCard" time="17.438" classname="scripts.CartOperTest"/>
  <testcase name="navigateToHomePage" time="2.521" classname="scripts.CartOperTest"/>
  <testcase name="echeckAddCartBtn" time="9.471" classname="scripts.CartOperTest"/>
  <testcase name="dcheckAddCartBtn" time="7.558" classname="scripts.CartOperTest"/>
  <testcase name="selectDistAndStockyard" time="14.589" classname="scripts.CartOperTest"/>
  <testcase name="checkAddCartBtn" time="5.601" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm" time="1.805" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm" time="1.700" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm" time="1.790" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm" time="1.802" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm" time="1.688" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm" time="1.786" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm" time="1.799" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm" time="1.879" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm" time="2.084" classname="scripts.CartOperTest"/>
  <testcase name="checkQtyB" time="0.093" classname="scripts.CartOperTest"/>
  <testcase name="checkCalculation" time="0.174" classname="scripts.CartOperTest"/>
  <testcase name="clickonAddToCart" time="4.083" classname="scripts.CartOperTest"/>
  <testcase name="checkStockyardInCart" time="2.093" classname="scripts.CartOperTest"/>
  <testcase name="checkQTYInCart" time="2.084" classname="scripts.CartOperTest"/>
  <testcase name="checkPriceInCart" time="2.107" classname="scripts.CartOperTest"/>
  <testcase name="clickOnBuy" time="0.129" classname="scripts.CartOperTest"/>
  <testcase name="checkDestination" time="5.638" classname="scripts.CartOperTest"/>
  <testcase name="checkDestinationWithOutMobile" time="8.311" classname="scripts.CartOperTest"/>
  <testcase name="checkDestinationWithInvalidmobile" time="9.803" classname="scripts.CartOperTest"/>
  <testcase name="checkDestinationWithvalidmobile" time="6.517" classname="scripts.CartOperTest"/>
  <testcase name="checkStockyardinCP" time="0.086" classname="scripts.CartOperTest"/>
  <testcase name="checkUnitPriceCP" time="0.048" classname="scripts.CartOperTest"/>
  <testcase name="checkTotalQTY" time="0.083" classname="scripts.CartOperTest"/>
  <testcase name="checkTotPriceCP" time="0.068" classname="scripts.CartOperTest"/>
  <testcase name="checkGSTinCP" time="0.050" classname="scripts.CartOperTest"/>
  <testcase name="checkTotalPayableinCP" time="0.068" classname="scripts.CartOperTest"/>
  <testcase name="navigateEazypay" time="10.514" classname="scripts.CartOperTest"/>
  <testcase name="checkprice" time="0.376" classname="scripts.CartOperTest"/>
  <testcase name="validatCan" time="15.480" classname="scripts.CartOperTest"/>
  <testcase name="checkResMsg" time="0.123" classname="scripts.CartOperTest"/>
  <testcase name="checkbookingid" time="0.057" classname="scripts.CartOperTest"/>
  <testcase name="checkstatus" time="0.060" classname="scripts.CartOperTest"/>
  <testcase name="checkaddress" time="0.056" classname="scripts.CartOperTest"/>
  <testcase name="navigateHm" time="5.165" classname="scripts.CartOperTest"/>
  <testcase name="preset" time="19.712" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm2" time="11.849" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm2" time="11.815" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm2" time="11.726" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm2" time="11.888" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm2" time="11.803" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm2" time="11.836" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm2" time="11.828" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm2" time="11.817" classname="scripts.CartOperTest"/>
  <testcase name="fillBookingForm2" time="11.876" classname="scripts.CartOperTest"/>
  <testcase name="checkAll" time="0.241" classname="scripts.CartOperTest"/>
  <testcase name="testaddcart" time="4.455" classname="scripts.CartOperTest"/>
  <testcase name="testQTY" time="2.089" classname="scripts.CartOperTest"/>
  <testcase name="testPrice" time="2.082" classname="scripts.CartOperTest"/>
  <testcase name="testA" time="0.151" classname="scripts.CartOperTest"/>
  <testcase name="RBwithoutLoca" time="5.595" classname="scripts.CartOperTest"/>
  <testcase name="rBWithoutAltmoba" time="8.316" classname="scripts.CartOperTest"/>
  <testcase name="rBInvalidMobilea" time="10.042" classname="scripts.CartOperTest"/>
  <testcase name="validatewithValidRBq" time="6.544" classname="scripts.CartOperTest"/>
  <testcase name="stockyardCPq" time="0.052" classname="scripts.CartOperTest"/>
  <testcase name="unitPriceCPs" time="0.032" classname="scripts.CartOperTest"/>
  <testcase name="totalQTYCPe" time="0.056" classname="scripts.CartOperTest"/>
  <testcase name="totalPriceinCPq" time="0.087" classname="scripts.CartOperTest"/>
  <testcase name="gstinCPd" time="0.083" classname="scripts.CartOperTest"/>
  <testcase name="totalPayableinCPg" time="0.066" classname="scripts.CartOperTest"/>
  <testcase name="navigateToPayments" time="10.527" classname="scripts.CartOperTest"/>
  <testcase name="paymentPrices" time="0.046" classname="scripts.CartOperTest"/>
  <testcase name="creditCncl" time="10.886" classname="scripts.CartOperTest"/>
  <testcase name="afterCredit" time="1.812" classname="scripts.CartOperTest"/>
  <testcase name="goHome" time="5.301" classname="scripts.CartOperTest"/>
</testsuite> <!-- scripts.CartOperTest -->
